steps/virtual-device-testing-for-android/0.9.1/step.yml

```diff
@@ -0,0 +1,165 @@
+title: '[PRIVATE-BETA] Virtual Device Testing for Android'
+summary: Run Android UI tests on virtual devices
+description: Run Android UI tests on virtual devices.
+website: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android
+source_code_url: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android
+support_url: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android/issues
+published_at: 2017-08-28T16:53:27.364922849+02:00
+source:
+  git: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android.git
+  commit: 11af5d56f7bce84c9642738c7b920b5c1be4f6e9
+host_os_tags:
+- ubuntu
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-virtual-device-testing-for-android
+deps:
+  brew:
+  - name: go
+  apt_get:
+  - name: golang
+    bin_name: go
+is_always_run: false
+is_skippable: false
+inputs:
+- apk_path: $BITRISE_APK_PATH
+  opts:
+    description: |
+      The path to the unaligned debug APK.
+    is_required: true
+    summary: The path to the debug, unaligned APK.
+    title: APK path
+- opts:
+    description: |
+      One device configuration per line and the parameters are separated with "," in the order of: deviceID, version, language, orientation
+      For example:
+      `NexusLowRes,24,en,portrait`
+      `NexusLowRes,24,en,landscape`
+    is_required: true
+    summary: |
+      One device configuration per line and the parameters are separated with "," in the order of: "deviceID, version, language, orientation". For example: `NexusLowRes,24,en,portrait`
+    title: Test devices
+  test_devices: NexusLowRes,24,en,portrait
+- opts:
+    description: |
+      The type of your test you want to run on the device.
+    is_required: true
+    summary: The type of your test
+    title: Test type
+    value_options:
+    - instrumentation
+    - robo
+    - gameloop
+  test_type: instrumentation
+- app_package_id: null
+  opts:
+    description: |
+      The package ID of your app
+    summary: The package ID of your apk
+    title: App package ID
+- opts:
+    category: Instrumentation Test
+    description: |
+      The path to the unaligned test debug APK.
```

Comment:
Mention that this apk should contain the instrumentation test runner, or explain it better